subsystems/viamagent/viam-agent.service

```diff
@@ -8,7 +8,7 @@ Type=exec
 Restart=always
 RestartSec=5
 User=root
-TimeoutSec=120
+TimeoutSec=240
```

Comment:
This should be modified to match in preinstall.sh (though it's not critical, as it'll update the first time it gets online anyway.)